src/ui/public/test_harness/test_harness.css

```diff
@@ -0,0 +1,17 @@
+body#test-harness-body {
```

Comment:
Maybe a note for why this is a CSS file? I assume it's because the test system needs it to be available outside of build?

Reply:
Haha. Fair enough. I'm guessing that's why.